### DISCUSSION TITLE
Anticipated review requests

### DISCUSSION THREAD

**Discussion by mmikhasenko** [ComPWA/polarimetry#173] [General]
- [x] add LS systematic uncert.
 - [x] quantify uncertainty due to grid discretization
 - [x] how the error polarization scales with the systematic uncert. of $\alpha$?

**Comment by redeboer** [comment]
Some remarks that already came by:
- Effect of correlations
- What do we mean by model-agnostic?
  - Why not just implement the full amplitude models?
  - (There was also a remark about a "weighted approach"?)

**Comment by redeboer** [comment]
Feedback/notes from meeting:
- Repeat statement on negligibility of:
  - detector effects
  - p.14 of original paper
  - (last uncertainty column from Table 8 ignored -- we could include this, that would make it a more conservative estimate)
- Provide explicit example of how to reconstruct -- use Li Yiming's example, B+ -> Lc+Lc- K+
- [Misha's worry]: Lack of correlations provided, how to better justify? [No answer]
- Approval meeting in 2-3 weeks -- it's a holiday in China beginning of October
- Minutes will be sent by Misha
